index.js

```diff
@@ -1,63 +1,67 @@
-// Make sure if the local storage in empty than add an empty array
-if (localStorage.getItem('Added Books') == null) {
-  localStorage.setItem('Added Books', JSON.stringify([]));
-}
+class BookCollection {
+  // Make sure if the local storage in empty than add an empty array
+  static getStoredBooks() {
```

Comment:
Good job was done so far 👍🏼  , I want just to mention that Static methods are often utility functions, such as functions to create or clone instances. Public static fields are useful when you want a field to exist only once per class, not on every class instance you create , here you need to have list of books as property of the class and it should be initialized in the constructor of the class in this case your BookCollection class can be reusable you can create different instances to manages different categories of books for example 
and onload you create a new instance of your BookCollection class

`constructor() {
    this.books = [];
  }
`

Reply:
@souad988 I'm confused about this part. Can you explain step by step on this?